.gitignore

```diff
@@ -1,4 +1,4 @@
-src/*.env
+src/.env
```

Comment:
.env is there for my local development process. there may be some more env like `test.env` so i need git to ignore all types of envs.

Reply:
note: if you mean example.env maybe rename it to `.env.example`?

Reply:
i more had it because of `example.env` but it should be `.env.example` (will change)